File: sioga/2.Codigo/backend/SiogaApiPublic/Domain/TipoDocumentoIdentidad.cs
using System;

namespace SiogaApiPublic.Domain
{
    public class TipoDocumentoIdentidad
    {
        public int TipoDocumentoIdentidadId { get; set; }
        public string Codigo { get; set; }
        public string Nombre { get; set; }
        public string Descripcion { get; set; }
        public bool Estado { get; set; }
    }
}

File: recaudacion/2.Codigo/frontend/src/app/core/models/tipodocumentoidentidad.ts
export class TipoDocumentoIdentidad {
    tipoDocumentoIdentidadId: number;
    codigo: string;
    nombre: string;
    descripcion: string;
    estado: boolean;
    usuarioCreador: string;
    usuarioModificador: string;
    index:number;
}
export class TipoDocumentoIdentidadFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn: string;
    sortOrder: string;
    nombre: string;
    estado: boolean;
}
